src/pages/SetPasswordPage.js

```diff
@@ -1,21 +1,22 @@
 import React, {Component} from 'react';
 import {
-    SafeAreaView,
     Text,
     TextInput,
     View,
+    SafeAreaView,
 } from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
 import _ from 'underscore';
 import lodashGet from 'lodash/get';
-import validateLinkPropTypes from './validateLinkPropTypes';
 import styles from '../styles/styles';
-import ExpensifyCashLogo from '../../assets/images/expensify-cash.svg';
 import {setPassword} from '../libs/actions/Session';
 import ONYXKEYS from '../ONYXKEYS';
-import variables from '../styles/variables';
 import ButtonWithLoader from '../components/ButtonWithLoader';
+import themeColors from '../styles/themes/default';
+import SignInPageLayout from './signin/SignInPageLayout';
+import canFocusInputOnScreenFocus from '../libs/canFocusInputOnScreenFocus';
+import validateLinkPropTypes from './validateLinkPropTypes';
```

Comment:
NAB, another example here e.g. this didn't _need_ to be moved from line 12 (and makes it slightly more difficult to tell whether we should consider this change or not). It's minor but just wanted to get you thinking about that stuff.

Reply:
Thanks @marcaaron! I need to be more careful when examining the diff, sorry for making this process more difficult that it needs to be. I really appreciate the feedback!